packages/engine/src/physics/functions/createCollider.ts

```diff
@@ -74,8 +76,16 @@ export const createShape = (entity: Entity, mesh: Mesh, shapeOptions: ShapeOptio
   // type is required
   if (!shapeOptions.type) return undefined!
 
-  const scale = mesh.scale ?? new Vector3(1, 1, 1)
+  // Get accurate scale relative to all parents
+  let scale = mesh.scale ?? new Vector3(1, 1, 1)
+  let currentChild = mesh as any
+  while (currentChild && currentChild.uuid !== Engine.scene.uuid) {
```

Comment:
we want to use object3d.traverseParents to apply this, and we may be able to update world matricies and use getWorldScale here

Reply:
Why didn't I think of that the first time😄.
Updated now.